larray/tests/test_misc.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import, division, print_function
+
+import pytest
+from larray import get_example_filepath
+
+
+def test_get_example_filepath():
+    with pytest.raises(ValueError, message="Example file non_existing_example_file.xlsx does not exist. "
```

Comment:
This will be annoying. We will need to change the test each time we add or rename an example file. It would be better to use a pattern to only check the beginning of the error message.